src/components/ui/QuestionOfTheDay.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  AccentText,
+  HeaderText,
+  QuestionOfDayText,
+  StatusText,
+} from 'components/ui/Text'
+import { useState } from 'preact/hooks'
+import Button from 'components/ui/Button'
+import Card from 'components/ui/Card'
+import CastBlock from 'components/Cast/CastBlock'
+import Delimiter from 'components/ui/Delimiter'
+import QuestionOfTheDayLabel from 'icons/QuestionOfTheDayLabel'
+import Sender from 'components/BlockchainList/Sender'
+import Thread from 'icons/Thread'
+import VertialDelimiter from 'components/ui/VertialDelimiter'
+import classnames, {
+  alignItems,
+  display,
+  flexDirection,
+  flexWrap,
+  gap,
+  margin,
+  position,
+  textColor,
+} from 'classnames/tailwind'
+
+const postInfo = classnames(
+  display('flex'),
+  flexDirection('flex-col', 'xs:flex-row'),
+  alignItems('items-baseline'),
+  flexWrap('flex-wrap'),
+  gap('gap-x-1')
+)
+
+const badgeWrapper = classnames(
+  position('absolute'),
+  margin('ml-4/5', '-mt-1/6')
```

Comment:
You have `absolute` there, why we need to use negative margin? Negative margin is pretty bad practice, we should do it only when there's no other way